javaparser-core/src/main/java/com/github/javaparser/ast/comments/Comment.java

```diff
@@ -61,7 +63,7 @@ public final String getContent() {
      */
     public Comment setContent(String content) {
         notifyPropertyChange(ObservableProperty.CONTENT, this.content, content);
-        this.content = content;
+        this.content = assertNotNull(content);
```

Comment:
maybe this assert not null should be used before notifying the property change, otherwise we risk notifying about a change that will not happen